cloudformation/membership-attribute-service.yaml

```diff
@@ -320,7 +309,8 @@ Resources:
   TopicGiraffe:
     Type: AWS::SNS::Topic
     Properties:
-      TopicName: giraffe
+      TopicName:
+        Fn::FindInMap: [ StageVariables, { Ref: Stage }, GiraffeTopic ]
```

Comment:
Just for notes, the removal of this stuff is in https://github.com/guardian/members-data-api/pull/131, which may need to resolve a merge conflict.